core/src/main/java/org/apache/spark/shuffle/sort/BypassMergeSortShuffleWriter.java

```diff
@@ -83,6 +84,7 @@
   private final int mapId;
   private final Serializer serializer;
   private final IndexShuffleBlockResolver shuffleBlockResolver;
+  private final int shuffleGenerationId;
```

Comment:
shall we move it right after `private final int shuffleId`?

Reply:
Thanks, done in 96bab1e.